lib/compiler/src/cerl_trees.erl

```diff
@@ -352,7 +352,7 @@ mapfold(F, S0, T) ->
 
 
 %% @spec mapfold(Pre, Post, Initial::term(), Tree::cerl()) -> {cerl(), term()}
-%%       Pre  = (cerl(), term()) -> {cerl(), term()}
+%%       Pre  = (cerl(), term()) -> {cerl(), term()} | skip
 %%       Post = (cerl(), term()) -> {cerl(), term()}
 %%
 %% @doc Does a combined map/fold operation on the nodes of the
```

Comment:
Please update the documentation to mention what `skip` means.

Reply:
Done!